pkg/registry/apis/provisioning/jobs/utils.go

```diff
@@ -1,34 +1,102 @@
 package jobs
 
-import "path"
+import (
```

Comment:
seems you are already doing some restructuring, I propose we move this `utils.go` to `resource` package as it helps to work with a the resources as a tree and it only contains this. It could be part of the `folder.go` file I mentioned. 

It doesn't have much to do with `utils` or `jobs` in my opinion but it's about the resources.

Reply:
I was working towards that goal to be completed as a separate PR, but will happily do it here instead 😄